src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/LoggerMessageGeneratorEmitterTests.cs

```diff
@@ -149,6 +149,20 @@ internal class Class2 { }
             await VerifyAgainstBaselineUsingFile("TestWithNestedClass.generated.txt", testSourceCode);
         }
 
+        [Fact]
+        public async Task TestBaseline_TestWithFileScopedNamespace_Success()
+        {
+            string testSourceCode = @"
+namespace Microsoft.Extensions.Logging.Generators.Tests.TestClasses;
+
+internal static partial class TestWithDefaultValues
+{
+    [LoggerMessage]
+    public static partial void M0(ILogger logger, LogLevel level);
+}";
+            await VerifyAgainstBaselineUsingFile("TestWithDefaultValues.generated.txt", testSourceCode);
```

Comment:
There's already another test here using `TestWithDefaultValues.generated.txt`, if you wanted to add one here, you need to use a new `*.generator.txt` file. And also add that file under the baseline folder.

Reply:
I just re-used it as the output is the same, but I can add another file with a different name with the same content.

Reply:
oh I see. ok then I didn't realize.